src/offlinelogstorage/dlt_offline_logstorage_behavior.c

```diff
@@ -371,20 +370,15 @@ int dlt_logstorage_storage_dir_info(DltLogStorageUserConfig *file_config,
         config->records = NULL;
     }
 
-    char suffix[10];
-    int suffix_len;
-    memset(suffix, 0, 10);
-    if (config->gzip_compression) {
-        suffix_len = DLT_OFFLINE_LOGSTORAGE_GZ_FILE_EXTENSION_LEN;
-        strncpy(suffix, ".dlt.gz", suffix_len);
-    }
-    else {
-        suffix_len = DLT_OFFLINE_LOGSTORAGE_FILE_EXTENSION_LEN;
```

Comment:
Please delete the two now unused defines from the header as well.

Reply:
Removed DLT_OFFLINE_LOGSTORAGE_FILE_EXTENSION_LEN